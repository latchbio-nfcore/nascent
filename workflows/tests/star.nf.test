nextflow_pipeline {

    name "Test Workflow main.nf with STAR"
    script "../../main.nf"
    tag "aligner"
    tag "star"

    test("Should run with STAR (default)") {

        when {
            params {
                outdir = "$outputDir"
                aligner = "star"
            }
        }

        then {
            assert workflow.success
            assert snapshot(
                workflow.trace.tasks().size(),
            ).match()
        }
    }
}
