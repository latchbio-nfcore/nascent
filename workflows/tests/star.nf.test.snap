{
    "Should run with STAR (default)": {
        "content": [
            139
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.03.0"
        },
        "timestamp": "2024-04-25T17:27:56.772645664"
    }
}